app/views/shared/_determinations_form.html.haml

```diff
@@ -63,7 +63,7 @@
               = b.check_box(checked: params[:state_reason].present? && params[:state_reason].include?(b.value), 'aria-labelledby': "checkbox-control-legend-1 #{b.text.to_css_class}")
               = b.label { b.text }
 
-          .form-group.panel.nested-fields.fieldset.spacer.js-reject-reason-text{ style: 'display: none; padding-top: 10px;', class: error_class?(@error_presenter, :rejected_reason_other) }
+          .form-group.panel.nested-fields.fieldset.spacer.js-reject-reason-text.hidden{ class: error_class?(@error_presenter, :rejected_reason_other) }
```

Comment:
Padding-top being removed...

Reply:
Yes, padding top was removed from the view, all styles now reside in the scss.